### PR TITLE
Add tests for loader signature validation

### PR DESCRIPTION
**Pull request by ameihm0912**
Adds tests for loader signature validation; this also resolves an issue related to duplicate signatures in the manifest.

### CODE REVIEW

File: mig-loader/loader_test.go [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+}`
+
+func bootstrap() {
+	ctx.Channels.Log = make(chan mig.Log, 0)
+	go func() {
+		for {
+			_ = <-ctx.Channels.Log
+		}
+	}()
+}
```
Comment (jvehent) [inline mig-loader/loader_test.go]: ❓ 🐰 
what does this do?

**Comment by jvehent** [comment]
r+ except for that weird bootstrap function that I can't understand

File: mig-loader/loader_test.go (hunk 1 above)
Comment (ameihm0912) [inline mig-loader/loader_test.go]: checkManifestSignature in the test functions logs, so this just sets up a routine to drain the logging channel so it doesn't block

Comment (jvehent) [inline mig-loader/loader_test.go]: Ha, that makes sense. May I suggest renaming it to `drainLogs` then?

**Pull request merged**